### ISSUE TITLE
Mismatch between result message and result image

### ISSUE THREAD

**Issue by yes89929** [CSAILVision/semantic-segmentation-pytorch#263]
I made custom dataset that has 3 classes.

And, I trained hrnet model with pretrained weights.
`python train.py --gpus 0,1,2 --cfg config/ade20k-hrnetv2.yaml`

Then, I validated a one image used training.

Output message is `Mean IoU: 1.0000, Accuracy: 100.00%`
![image](https://user-images.githubusercontent.com/33365390/126595512-7474cd3b-078c-4d89-8065-99279cd35367.png)

But, when looking result image, it's not.
![image](https://user-images.githubusercontent.com/33365390/126595730-22e6f7fe-76e1-478b-8556-139b0af464c0.png)

I printed number of  pixcel in intersection and union.
And, number of gray pixcel is 12614.
However, it is must be > 1,000,000, considering result image.

![image](https://user-images.githubusercontent.com/33365390/126596232-9ff57453-e1f1-49ca-a088-c4039998016e.png)
`### intersection:  [12614  4295  3970] union [12614  4295  3970]`

I have no idea what is problem.
